com.github.zaps166.QMPlay2.json

```diff
@@ -0,0 +1,150 @@
+{
+  "app-id": "com.github.zaps166.QMPlay2",
```

Comment:
```suggestion
  "app-id": "io.github.zaps166.QMPlay2",
```

Reply:
wrong assumption. It has always been the case it's now strictly enforced.

See:
https://docs.flatpak.org/en/latest/conventions.html#application-ids